Tasks/XamariniOSV2/Tests/L0BuildToolLocation.ts

```diff
@@ -0,0 +1,67 @@
+import ma = require('vsts-task-lib/mock-answer');
+import tmrm = require('vsts-task-lib/mock-run');
+import path = require('path');
+import os = require('os');
+
+let taskPath = path.join(__dirname, '..', 'xamarinios.js');
+let tr: tmrm.TaskMockRunner = new tmrm.TaskMockRunner(taskPath);
+
+process.env['HOME']='/user/home'; //replace with mock of setVariable when task-lib has the support
+
+// Required inputs
+tr.setInput('solution', 'src/project.sln'); //path
+tr.setInput('configuration', 'Release');
+// Optional inputs
+tr.setInput('args', '');
+tr.setInput('packageApp', ''); //boolean
+tr.setInput('forSimulator', ''); //boolean
+tr.setInput('buildToolLocation', '/home/bin2/msbuild');
+tr.setInput('runNugetRestore', 'true'); //boolean
+tr.setInput('iosSigningIdentity', '');
+tr.setInput('provProfileUuid', '');
+
+// provide answers for task mock
+let a: ma.TaskLibAnswers = <ma.TaskLibAnswers>{
```

Comment:
I made a change to `TaskLibAnswers` so we shouldn't have to type assert anymore.  Removing the type assertion will catch mocks that don't work (`getVariable` in this case).

Reply:
This task in on older version of the lib!